cpp/tests/test_odesecirvvs.cpp

```diff
@@ -1178,13 +1174,53 @@ TEST(TestOdeSECIRVVS, apply_constraints_parameters)
     EXPECT_EQ(model.parameters.apply_constraints(), 1);
     EXPECT_EQ(model.parameters.get<mio::osecirvvs::ReducTimeInfectedMild>()[indx_agegroup], 1);
 
-    model.parameters.set<mio::osecirvvs::BaseInfectiousnessB117>(-0.5);
-    EXPECT_EQ(model.parameters.apply_constraints(), 1);
-    EXPECT_EQ(model.parameters.get<mio::osecirvvs::BaseInfectiousnessB117>()[indx_agegroup], 0);
-
-    model.parameters.set<mio::osecirvvs::BaseInfectiousnessB161>(-4);
+    model.parameters.set<mio::osecirvvs::InfectiousnessNewVariant>(-4);
     EXPECT_EQ(model.parameters.apply_constraints(), 1);
-    EXPECT_EQ(model.parameters.get<mio::osecirvvs::BaseInfectiousnessB161>()[indx_agegroup], 0);
+    EXPECT_EQ(model.parameters.get<mio::osecirvvs::InfectiousnessNewVariant>()[indx_agegroup], 1);
 
     mio::set_log_level(mio::LogLevel::warn);
 }
+
+TEST(TestOdeSECIRVVS, apply_variant_function)
```

Comment:
Consider making comments on what this test is supposed to do, where the numbers com from, etc